tests/testthat/test-tabPanel.R

```diff
@@ -0,0 +1,14 @@
+
+context("tabPanel")
```

Comment:
Modern practice is to omit `context()`

Reply:
I ran into a situation with `shiny::shinyAppTemplate()`... if no file had a context, then the errors were not printed.  By adding the context, then errors were reported correctly.